apps/files_external/src/actions/inlineStorageCheckAction.ts

```diff
@@ -34,45 +34,51 @@ export const action = new FileAction({
 	 * @param node The node to render inline
 	 */
 	async renderInline(node: Node) {
+		const span = document.createElement('span')
+		span.className = 'files-list__row-status'
+		span.innerHTML = t('files_external', 'Checking storage...')
```

Comment:
```suggestion
		span.innerHTML = t('files_external', 'Checking storage …')
```

- https://github.com/nextcloud/server/pull/50946